lib/active_fedora/associations/belongs_to_association.rb

```diff
@@ -36,16 +36,6 @@ def remove_keys
           owner[reflection.foreign_key] = nil
         end
 
-        # Constructs a query that checks solr for the correct id & class_name combination
-        def construct_query(ids)
```

Comment:
It should have been removed here: https://github.com/projecthydra/active_fedora/commit/bbf94f4354b83f0e3463b0ab19e3bb86b63203e8